New page to document Asset Processor and Asset Builder Settings (#1642)
o3de/o3de.org: content/docs/user-guide/assets/asset-processor/debugging.md

---
linkTitle: Debugging
title: Debugging Asset Processor Issues
description: Methods to debug Asset Processor issues in Open 3D Engine (O3DE).
weight: 500
toc: true
---

If you are having issues with **Asset Processor**, this page covers common problems, solutions, and debugging techniques.

Some of the most common problems we've seen with Asset Processor include:

> * After changing a source asset, it takes too long for assets to finish processing.
> * The product assets don't behave like expected. They may be missing entirely, missing information, have incorrect information.

Reviewer: ```suggestion
* Asset processing takes too long after changes are made to a source asset.
* The product assets don't behave like expected. They may have incorrect information, be missing information, or be missing entirely.
```

* Unexpected warnings or errors.

> * Asset packaging isn't working as expected, content is missing or content is being included in bundles that was not expected to be there.

Reviewer: ```suggestion
* Asset packaging isn't working as expected. Content is missing or content is being included in bundles that was not expected to be there.
```
